sklearn/utils/validation.py

```diff
@@ -1181,7 +1181,6 @@ def has_fit_parameter(estimator, parameter):
     >>> from sklearn.utils.validation import has_fit_parameter
     >>> has_fit_parameter(SVC(), "sample_weight")
     True
-
     """
     return parameter in signature(estimator.fit).parameters
```

Comment:
It is missing a space before `:` for the parameter `is_parameter`.

```
is_parameter : bool
```

Reply:
fixed